buildSrc/settings.gradle.kts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    // Add support for `libs.versions.toml` within `buildSrc`
+    // https://github.com/radoslaw-panuszewski/typesafe-conventions-gradle-plugin
+    // https://github.com/gradle/gradle/issues/15383
+    id("dev.panuszewski.typesafe-conventions") version "0.4.1"
+}
```

Comment:
We might well simply put the following content to this file to reuse the toml file in the main build (which seems a lightweight way without a third-party gradle plugin)
```
dependencyResolutionManagement {
    versionCatalogs {
        create("libs") {
            from(files("../gradle/libs.versions.toml"))
        }
    }
}
```